utils/build.ps1

```diff
@@ -1247,6 +1395,7 @@ function Build-ICU([Platform]$Platform, $Arch) {
     # Use previously built x64 tools
     $BuildToolsDefines = @{
       BUILD_TOOLS = "NO";
+      BUILD_DATA = if ($Platform -eq "Android") { "NO" } else { "YES" };
```

Comment:
Crazy idea: would could assign booleans here and have `Build-CMakeProject` convert them into `"YES"/"NO"`

Reply:
Eh, I consider outside the scope of this change, but I don't have a problem with that.